kubernetes/cray-kafka-operator/values.yaml

```diff
@@ -7,10 +7,77 @@ strimzi-kafka-operator:
     - sma
 
   image:
-    repository: artifactory.algol60.net/csm-docker/stable/quay.io/strimzi
+    repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
     name: operator
     tag: 0.15.0
     imagePullPolicy: IfNotPresent
+
+  zookeeper:
+    image:
+      respository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
```

Comment:
respository -> repository
this applies to all these settings.

Reply:
I must have fat-fingered the search/replace. Fixed.